src/com/sun/ts/tests/signaturetest/README

```diff
@@ -57,7 +57,7 @@ Users should see if their signatures already exist in this directory.
 Note, every property in the sig-test.map file maps to a signature
 file.  For example the property "javax.ejb=2.1" maps to the signature
 file:
- $TS_HOME/src/com/sun/ts/tests/signaturetest/signature-repository/javax.ejb.sig_2.1
+ $TS_HOME/src/com/sun/ts/tests/signaturetest/signature-repository/jakarta.ejb.sig_2.1
```

Comment:
I think that this should be jakarta.ejb.sig_3.2_se11 or jakarta.ejb.sig_3.2_se8, instead of referring to 2.1

Reply:
Thanks, I updated to jakarta.ejb.sig_3.2_se11 in 4a4946e